src/dotnet-symbol/Program.cs

```diff
@@ -355,9 +375,9 @@ private async Task WriteFileToDirectory(Stream stream, string fileName, string d
         {
             stream.Position = 0;
             string destination = Path.Combine(destinationDirectory, Path.GetFileName(fileName.Replace('\\', '/')));
-            if (File.Exists(destination))
+            if (!Overwrite && File.Exists(destination))
```

Comment:
Two things I never considered about this:

- This won't work well for things like ALCs, where multiple versions of a dll with the same name will end up with symbol collisions.
- With this flag, we should pull this logic up to `DownloadFiles` and only download the file if (Overwrite || !File.Exists(key.FullPathName))

Reply:
Since the default is still NOT to overwrite, your first point isn't any worse. If we have name collisions then the first one wins which is bad (already bad).

Your second point is a good idea. I'll try it out.

Reply:
Never mind about the second part.  It would involve too many changes and refactor to get the "destination" file path in download.  I hope it is ok to leave it be.